src/chi/javascript/components/dropdown.js

```diff
@@ -61,6 +61,9 @@ class Dropdown extends Component {
 
     this._dropdownElemClickEventListener = function(e) {
       self._dropdownClickedEventManager(e);
+      if (e.target.tagName.toLowerCase() === 'a' || e.target.tagName.toLowerCase() === 'button') {
```

Comment:
Please, use nodeName instead of tagName for consistency. There's also no need to use toLowerCase. Spec says this is the name of the tag in capital letters so you can check against 'A' or 'BUTTON' directly.